src/App.js

```diff
@@ -9,7 +10,14 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <div>
+    <h1>React Article</h1>
+    <Article
+      title={article.title}
+      text={article.text}
+      date={article.date}
+    />
```

Comment:
можно указать явно, можно распаковать
```suggestion
    <Article {...article} />
```